tests/unit/plugins/test_base.py

```diff
@@ -63,7 +63,7 @@ def test_plugin(new_dir):
     props = FooPluginProperties.unmarshal({"foo-name": "world"})
     plugin = FooPlugin(properties=props, part_info=part_info)
 
-    assert isinstance(plugin.properties_class, PluginProperties)
+    assert plugin.properties_class == FooPluginProperties
```

Comment:
You define it above, so I wouldn't even bother with the test (unless PluginProperties writes to it).

```suggestion
```